jenkins/tippecanoe_create_tiles_Jenkinsfile

```diff
@@ -22,12 +23,13 @@ pipeline {
       }
       steps {
         sh 'tippecanoe -Z4 -z8 --no-tiny-polygon-reduction --detect-shared-borders --simplify-only-low-zooms --simplification=5 --force --output-to-directory tile_dir_simple5 hrus.geojson'
+        sh 'tile-join --output-to-directory tile_dir_simple5_data -c hru_data.csv tile_dir_simple5' 
       }
     }
     stage('push to S3') {
       agent any
       steps { 
-        sh 'aws s3 sync tile_dir_simple5 s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/tile_dir_simple5'
+        sh 'aws s3 sync tile_dir_simple5_data s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/tile_dir_simple5_data'
```

Comment:
@abriggs-usgs would it be useful to start shipping the tiles joined with data out to where they'll actually be served from instead of here?

Reply:
Ya. I would say that we can just skip the data bucket (or use the data bucket for versioning) and deliver the joined tiles directly to the 'test' and 'beta' buckets.

Reply:
If we are going to have a separate job to do the tile-join, that will be the one that sends the finished tiles out to the appropriate bucket, so I'd say we can just leave this one pushing to the common resources bucket.